Сетка страниц и крупных блоков на флексах

Iliukich opened the pull request:
---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/36/user/2259269/tasks/10)

:boom: https://htmlacademy-htmlcss.github.io/2259269-gllacy-36/5/

Iliukich committed:
Добавлены крупные сетки

SineYlo commented on project/index.html:
```diff
@@ -1,288 +1,324 @@
+              <dd class="contacts-desription">
+                <a class="contacts-phone" href="tel:+78005558628">+7 800 555-86-28</a>
+              </dd>
+              <dd class="contacts-desription"><span class="contacts-inscription">(с 10 до 20 ежедневно)</span></dd>
```
Данный элемент можно убрать, а содержимое добавить к предыдущему `<dd>`
У каждого `<dd>` должен быть `<dt>`, а тут его нет

SineYlo reviewed (commented):
В целом всё неплохо, но я заметил ещё 2 странных момента:
1. Почему-то картинка в секции с контактами плохо отображается, даже если применить `background-position: center`. Возможно, что вы как-то некорректно её выгрузили или ещё что-то. Перепроверьте данный момент.
2. У вас отсутствует элемент с классом `container`, который как раз таки бы и центрировал всё содержимое, а сейчас у вас получается так, что вы задаёте это для каждой секции. По-хорошему сразу внутри шапки, подвала и секций должен быть `container` с шириной и `margin: 0 auto`. Можно ещё добавить паддинги, но тогда нужно будет пересчитать ширину с их учётом. Например, если паддинги по `15px`, то итоговая ширина уже будет `1200px`, а не `1170px`.
— — —
Также напоминаю, что у вас осталось довольно мало времени до завершения курса. Если будете понимать, что прям совсем не успеваете, то можете задействовать доступные 9 часов и мы решим вместе оставшиеся задания.

Iliukich committed:
Исправлены некоторые ошибки

Убрал в некоторых местах не меняющие ничего ширину и margin, изменена картинка(оригинал картинки в макете весил 4 мегабайта и имел слишком большое разрешение для того, чтобы поместиться правильно в блок.). Немного не понял второй пункт. Имеется ввиду обернуть всё содержимое страницы в один контейнер?
